File: CSharp/OpenTraceability.Tests/Events/UtilityTests.cs
using System;
namespace OpenTraceability.Tests.Events
{
	[TestFixture]
	public class UtilityTests
	{
		[Test]
		[TestCase("EPCISQueryDocument.jsonld")]
		public void JSONCompareTest(string file)
		{
			string json1 = OpenTraceabilityTests.ReadTestData(file);
			string json2 = OpenTraceabilityTests.ReadTestData(file);
			OpenTraceabilityTests.CompareJSON(json1, json2);
        }
	}
}

File: Java/OpenTraceability/src/test/kotlin/opentraceability/event/UtilityTests.kt
package opentraceability.event

import opentraceability.OpenTraceabilityTests
import org.junit.jupiter.api.Test

class UtilityTests {
    @Test
    //@ParameterizedTest(name = "{0}")
    //@ValueSource(strings = ["EPCISQueryDocument.jsonld"])
    fun jsonCompareTest() {
        val json1 = OpenTraceabilityTests.readTestData("EPCISQueryDocument.jsonld")
        val json2 = OpenTraceabilityTests.readTestData("EPCISQueryDocument.jsonld")
        OpenTraceabilityTests.compareJSON(json1, json2)
    }
}
